CHANGELOG.md

```diff
@@ -22,7 +22,8 @@ These are the section headers that we use:
 - Added `HuggingFaceDatasetMixIn` for internal usage, to detach the `FeedbackDataset` integrations from the class itself, and use MixIns instead ([#3326](https://github.com/argilla-io/argilla/pull/3326)).
 - Added `GET /api/v1/records/{record_id}/suggestions` API endpoint to get the list of suggestions for the responses associated to a record ([#3304](https://github.com/argilla-io/argilla/pull/3304)).
 - Added `POST /api/v1/records/{record_id}/suggestions` API endpoint to create a suggestion for a response associated to a record ([#3304](https://github.com/argilla-io/argilla/pull/3304)).
-- Added breaking simutaneously running tests within GitHub package worflows. ([#3354](https://github.com/argilla-io/argilla/pull/3354)).
+- Added breaking simultaneously running tests within GitHub package workflows ([#3354](https://github.com/argilla-io/argilla/pull/3354))
+- Added API and Python Client support for workspace deletion (Closes [#3260](https://github.com/argilla-io/argilla/issues/3260))
```

Comment:
I guess here we should point to the PRs right?

```suggestion
- Added API and Python Client support for workspace deletion ([#3358](https://github.com/argilla-io/argilla/pull/3358))
```

Reply:
Nope, if an issue is created we should point out the issue itself. The issue should be then properly linked to the PR so users can walk through the changes.

Reply:
Cool, then I'll do that from now on I guess I was always pointing to PRs as it was easier to see the actual code change 😄